backend/howtheyvote/api/votes_api.py

```diff
@@ -436,6 +441,39 @@ def _format_sources(fragments: Iterable[Fragment]) -> list[SourceDict]:
     return sorted(sources, key=lambda s: s["accessed_at"])
 
 
+def _format_links(vote: Vote) -> list[LinkDict]:
+    links: list[LinkDict] = []
+
+    if vote.press_release:
+        links.append(
+            {
+                "title": "Press release",
+                "description": "Press release published by the European Parliament’s Press Service. Press releases often contain more information about the subject of the vote and next steps.",  #  noqa: E501
+                "url": press_release_url(vote.press_release),
+            }
+        )
+
+    if vote.reference:
+        links.append(
+            {
+                "title": "Report or resolution",
+                "description": "Original text of the report or resolution as tabled. The text may be different from the adopted text if MEPs have adopted amendments.",  #  noqa: E501
+                "url": doceo_document_url(vote.reference),
+            }
+        )
```

Comment:
We do not need to do this now, but I think we should consider the following:
In the cases where amendments have been adopted, the resolution page will often (I would hope always, but who knows) display a link to the text as it was adopted in the "flow-chart" under "Texts adopted". Scraping that as well would of course increase complexity, but make contents of the link we provide far more intuitive.
The link is also retrievable from OEIL.